tests/testthat/test-qag.R

```diff
@@ -43,11 +43,11 @@ test_that("Integration agrees with R on simple problem", {
 
   ## These should all be different, though the first case is not
   ## necessarily so.
-  ## expect_false(identical(int_15$last_area, int_21$last_area))
+  expect_false(identical(int_15$last_area, int_21$last_area))
   expect_false(identical(int_21$last_area, int_31$last_area))
   expect_false(identical(int_31$last_area, int_41$last_area))
   expect_false(identical(int_41$last_area, int_51$last_area))
-  expect_false(identical(int_51$last_area, int_61$last_area))
+  #expect_false(identical(int_51$last_area, int_61$last_area))
```

Comment:
This seems to be failing: [link to GHA](https://github.com/traitecoevo/plant/actions/runs/5064809599/jobs/9092754628?pr=367#step:6:312)

```r
 ══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-qag.R:46:3'): Integration agrees with R on simple problem ────
identical(int_15$last_area, int_21$last_area) is not FALSE

`actual`:   TRUE 
`expected`: FALSE
── Failure ('test-qk.R:39:3'): Integration agrees with R on simple problem ─────
identical(int_15$last_area, int_21$last_area) is not FALSE

`actual`:   TRUE 
`expected`: FALSE

[ FAIL 2 | WARN 3 | SKIP 0 | PASS 2204 ]
```